src/parser/monk/shared/modules/conduits/FortifyingIngredients.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import ItemHealingDone from 'interface/ItemHealingDone';
+
+class FortifyingIngredients extends Analyzer {
+
+  shield = 0;
+
+  /**
+   * Whenever you cast fort brew you gain a shield for x% of your hp
+   */
+  constructor(...args) {
+    super(...args);
+    this.active = true;//this.selectedCombatant.hasConduit(SPELLS.FORTIFYING_INGREDIENTS.id);
```

Comment:
this condition should be set based on `hasConduit`. (I assume that function exists but will always return false?)

Reply:
The function doesn't exist but this is what I assume it will be. It also allows me to have most of this pretyped... i should have theses set to false though...